[website] Update the careers's page with the new roles (#32535)
mui/material-ui: docs/src/pages/careers/engineering-manager.md

# Engineering Manager - Toolpad

> <p class="description">You will grow the engineering team working on MUI Toolpad, from a small team to multiple.</p>

Reviewer: ```suggestion
<p class="description">You will grow the small engineering team currently working on MUI Toolpad.</p>
```

For additional details about the MUI team and culture, you can check our [careers](https://mui.com/careers/) and [about](/about/) pages and also our [public Handbook](https://mui-org.notion.site/Handbook-f086d47e10794d5e839aef9dc67f324b).

## Why we're hiring

Both our open-source community and our premium products are growing fast (x2-3 YoY).
We need talented people to keep that going!

Our mission is to empower as many people as possible to build great UIs, faster.
The faster and simpler it is, and the broader the audience that can create custom UIs, the better.
We believe that the best way to improve on these dimensions is to eliminate [80%](https://www.youtube.com/watch?v=GnO7D5UaDig&t=2451s) of the code that has to be written.

A few months back we started to work on [MUI Toolpad](https://github.com/mui/mui-toolpad), an ambitious project to deliver on this objective.
We have found the beginning of a market fit in this low-code segment.
We need help to structure & grow the engineering team.

### Why this is interesting

Our solution empowers React developers to build awesome applications faster – we see over a million developers on MUI's docs every month.

> Toolpad targets ~1/2 of this audience. While the team is still very small (5), we believe that the headcount on this product has the potential to grow +14% MoM until the end of 2023 (>70 people).

Reviewer: ```suggestion
Toolpad targets around half of this audience. While the team is still very small (5), we believe that the headcount on this product has needs to grow +14% MoM until the end of 2023 (>70 people).
```

> Growing the engineering side will be full of challenges.

Reviewer: ```suggestion
Growing the Toolpad engineering team will be full of challenges.
```

- Improve our processes, e.g. the lifecycle of feature development from design through testing and release.

## About you

### Skills you should have

- You've managed or supported a diverse, distributed engineers on small or large teams.
- You have an interest in building a support structure for engineers to help them succeed.
- You have phenomenal communication skills for working across the organization, capable of building strong relationships with peers and leadership.
- You have had strong expertise and mastery of React or front-end development in the past.
- You have deep empathy for users. You have experience with customer support or OSS community support.

### What would be nice if you had, but isn't required

- **You've maintained an active repository before**. Maybe you've helped maintain a popular open-source repository, or perhaps you've worked on internal repositories that saw contributions from multiple teams. Previous experience with highly active repository workflows is a definite plus for this role.

## Benefits & Compensation

Competitive compensation depending on the profile and location.

> We are ready to pay top market rates for a person that can significantly push the mission forward.

Reviewer: > Push the mission forward.

Anything but this! 🤮  🤣

Author: Alright, let's make it more concrete then.
